src/admin-creation.php

```diff
@@ -1,119 +1,147 @@
-<!DOCTYPE html>
-<html lang="en">
-
-<head>
-	<meta charset="UTF-8" />
-	<meta http-equiv="X-UA-Compatible" content="IE=edge" />
-	<meta name="viewport" content="width=device-width, initial-scale=1.0" />
-	<link rel="icon" type="image/x-icon" href="images/resc/ivote-favicon.png">
-	<title>Admin Creation</title>
-
-	<!-- Icons -->
-	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css" />
-	<script src="https://cdn.jsdelivr.net/npm/feather-icons/dist/feather.min.js"></script>
-
-	<!-- UPON USE, REMOVE/CHANGE THE '../../' -->
-    <link rel="stylesheet" href="styles/styles.css" />
-	<link rel="stylesheet" href="styles/admin-creation.css" />
-	<link rel="stylesheet" href="../vendor/node_modules/bootstrap/dist/css/bootstrap.min.css" />
-
-</head>
-
-<body>
-
-	<!---------- SIDEBAR + HEADER START ------------>
-	<?php include_once __DIR__ . '/includes/components/sidebar.php'; ?>
-	<!---------- SIDEBAR + HEADER END ------------>
-
+<?php
+require_once 'includes/classes/db-connector.php';
+require_once 'includes/session-handler.php';
```

Comment:
Kindly use the `normalizeFilepath` function for **includes** and **requires**

Here is the sample
![image](https://github.com/BSIT-3-1-APPDEV/PUPSRC-AutomatedElectionSystem/assets/92857477/f31202a7-3f8d-4859-a2ea-49942a4fd909)

the function is available if you include the file utility `class include_once str_replace('/', DIRECTORY_SEPARATOR, 'includes/classes/file-utils.php');` _**file path may vary**_

The purpose is that we are not sure of what operating system our website would be hosted. The operating system varies on directory separators read more [here](https://en.wikipedia.org/wiki/Path_(computing)#:~:text=paths%20are%20appended.-,Representations%20of%20paths%20by%20operating%20system%20and%20shell,-%5Bedit%5D)
This function appropriately assigns the correct directory separator for the current OS. It's better to be safe thank you.